lib/static/new-ui/components/MetaInfo/index.tsx

```diff
@@ -109,6 +110,23 @@ function MetaInfoInternal(props: MetaInfoInternalProps): ReactNode {
         });
     }
 
+    const shouldAddSkipReason = Boolean(!metaInfoItemsWithResolvedUrls.find(item => item.label === 'muteReason'));
```

Comment:
I would use a separate variable for this expression - `Boolean(!metaInfoItemsWithResolvedUrls.find(item => item.label === 'muteReason'))`. For example - `hasMuteReason`

Reply:
We discussed this with sipayrt and decided that it's not very good idea to include such specific logic related to `muteReason`, so i got rid of altogether